specification/purview/Azure.Analytics.Purview.DataMap/routes.tsp

```diff
@@ -0,0 +1,2245 @@
+""")
+      @query
+      isOverwrite?: boolean;
+
+      #suppress "@azure-tools/typespec-azure-core/bad-record-type" "Should use unknown to model Object"
+      @doc("BusinessMetadata payload")
+      @body
+      businessMetadata: Record<Record<unknown>>;
+    },
+    void,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/bad-record-type" "Should use unknown to model Object"
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is an Atlas existing api"
+  @doc("Delete business metadata attributes from an entity.")
+  @route("/atlas/v2/entity/guid/{guid}/businessmetadata/{bmName}")
+  @delete
+  removeBusinessMetadataAttributes is Azure.Core.Foundations.Operation<
+    {
+      @doc("BusinessMetadata name")
+      @path
+      bmName: string;
+
+      @doc("The globally unique identifier of the entity.")
+      @path
+      guid: string;
+
+      #suppress "@azure-tools/typespec-azure-core/bad-record-type" "Should use unknown to model Object"
+      @doc("Business metadata attribute payload")
+      @body
+      businessMetadataAttributes: Record<unknown>;
+    },
+    void,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is an Atlas existing api"
+  #suppress "@azure-tools/typespec-azure-core/bad-record-type" "Should use unknown to model Object"
+  @doc("Add or update business metadata attributes.")
+  @route("/atlas/v2/entity/guid/{guid}/businessmetadata/{bmName}")
+  @post
+  addOrUpdateBusinessMetadataAttributes is Azure.Core.Foundations.Operation<
+    {
+      @doc("BusinessMetadata name")
+      @path
+      bmName: string;
+
+      @doc("The globally unique identifier of the entity.")
+      @path
+      guid: string;
+
+      #suppress "@azure-tools/typespec-azure-core/bad-record-type" "Should use unknown to model Object"
+      @doc("Business metadata attribute payload")
+      @body
+      businessMetadataAttributes: Record<unknown>;
+    },
+    void,
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("Get the sample Template for uploading/creating bulk BusinessMetaData")
+  @route("/atlas/v2/entity/businessmetadata/import/template")
+  @get
+  getSampleBusinessMetadataTemplate is Azure.Core.RpcOperation<
+    {},
+    {
+      @header contentType: "application/octet-stream";
+      @body template: bytes;
+    },
+    {},
+    AtlasErrorResponse
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/byos" "representation of existing multipart/form-data operation"
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("Upload the file for creating Business Metadata in BULK")
+  @route("/atlas/v2/entity/businessmetadata/import")
+  @post
+  importBusinessMetadata is AtlasImportOperation<
+    {
+      @doc("InputStream of file")
+      uploadedInputStream: bytes;
```

Comment:
`multipart/form-data` is usually for scenario where payload is mixed with bytes and string. So if you only want to upload bytes, there is no need to use multipart. Of course, if this service plan to add other part type in payload, it is ok to define it in advance.

Reply:
Thanks for the information. This is one of the Atlas API https://atlas.apache.org/api/v2/resource_EntityREST.html#resource_EntityREST_importBMAttributes_POST. So we keep it as multipart/form-data.

Reply:
At present, Java won't handle the anonymous model on multipart. Next release would just have it a model `importBusinessMetadataRequest`.